src/utils/task_stats_alloc/tests/loom.rs

```diff
@@ -0,0 +1,101 @@
+#![feature(allocator_api)]
+#![cfg(loom)]
+
+// use task_stats_alloc::*;
+use std::alloc::System;
+use std::borrow::BorrowMut;
+use std::hint::black_box;
+use std::ptr::NonNull;
+use std::time::Duration;
+
+use loom::sync::atomic::{fence, AtomicUsize, Ordering};
+use loom::sync::Arc;
+use loom::thread;
+use task_stats_alloc::{allocation_stat, BYTES_ALLOCATED};
+use tokio::runtime::Handle;
+
+#[repr(transparent)]
+#[derive(Clone, Debug)]
+pub struct TaskLocalBytesAllocated(Option<NonNull<AtomicUsize>>);
```

Comment:
Is it possible to directly use the implementation from lib.rs and add `#[cfg]` to switch dependencies?

```rust
#[cfg(loom)]
use loom::sync::Arc;
#[cfg(not(loom))]
use std::sync::Arc;
```

Reply:
Yes it's possible. 

I plan to do this when we re-use `TaskLocalBytesAllocated` in the lib. Here this one is just a copy 😆